framework/src/modules/dpos_v2/utils.ts

```diff
@@ -249,3 +249,15 @@ export function getModuleConfig(config: ModuleConfigJSON): ModuleConfig {
 		tokenIDDPoS: Buffer.from(config.tokenIDDPoS, 'hex'),
 	};
 }
+
+export const getDelegateWeight = (
+	factorSelfVotes: bigint,
+	selfVotes: bigint,
+	totalVotesReceived: bigint,
+) => {
+	const cap = selfVotes * BigInt(factorSelfVotes);
```

Comment:
`factorSelfVotes` is already a bigint. What is the reason to explicitly cast it using `BigInt()`?